src/Silverback.Integration/Messaging/Messages/DefaultMessageHeaders.cs

```diff
@@ -99,5 +99,41 @@ public static class DefaultMessageHeaders
         ///     consumer side to determine the correct key to be used to decrypt the message.
         /// </remarks>
         public const string EncryptionKeyId = "x-encryption-key-id";
+
+        /// <summary>
+        ///     This will be set by the Move error policy and will contain the name of the consumer who processed the
+        ///     failed message.
+        /// </summary>
+        public const string SourceConsumer = "x-source-consumer";
+
+        /// <summary>
+        ///     This will be set by the Move error policy and will contain the reason the failed
+        ///     message is being moved.
+        /// </summary>
+        public const string InvalidReason = "x-invalid-reason";
```

Comment:
Yeah, I don't like this. It should be something like `x-failure-reason` or `x-processing-error`.

Reply:
Renamed and moved to `KafkaMessageHeaders`.